source/core/ib_particle.cc

```diff
@@ -108,16 +108,16 @@ IBParticle<dim>::get_properties_name()
   properties[PropertiesIndex::vx] = std::make_pair("Velocity", 3);
   properties[PropertiesIndex::vy] = std::make_pair("Velocity", 1);
   properties[PropertiesIndex::vz] = std::make_pair("Velocity", 1);
-  properties[PropertiesIndex::fx] = std::make_pair("Force", 3);
-  properties[PropertiesIndex::fy] = std::make_pair("Force", 1);
-  properties[PropertiesIndex::fz] = std::make_pair("Force", 1);
-  properties[PropertiesIndex::m]  = std::make_pair("Mass", 1);
   properties[PropertiesIndex::ox] = std::make_pair("Omega", 3);
   properties[PropertiesIndex::oy] = std::make_pair("Omega", 1);
   properties[PropertiesIndex::oz] = std::make_pair("Omega", 1);
+  properties[PropertiesIndex::fx] = std::make_pair("Force", 3);
+  properties[PropertiesIndex::fy] = std::make_pair("Force", 1);
+  properties[PropertiesIndex::fz] = std::make_pair("Force", 1);
   properties[PropertiesIndex::tx] = std::make_pair("Torques", 3);
   properties[PropertiesIndex::ty] = std::make_pair("Torques", 1);
   properties[PropertiesIndex::tz] = std::make_pair("Torques", 1);
+  properties[PropertiesIndex::m]  = std::make_pair("Mass", 1);
```

Comment:
I would use this opportunity to remove the capital letters. Regular Lethe does not use Capital letters for variables in paraview, let's keep it the same everywhere